build.sh

```diff
@@ -15,7 +15,11 @@ export DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1
 export DOTNET_MULTILEVEL_LOOKUP=0
 export NUGET_PACKAGES="$SCRIPT_ROOT/packages/"
 
-"$SCRIPT_ROOT/init-tools.sh"
+(
+    # init-tools.sh isn't compatible with these flags. Unset them in this subshell.
+    set +euo pipefail
```

Comment:
CI failed and I had to add this workaround, still look ok @weshaggard?

Reply:
Can we fix init-tools so it works with this option? What was the failure?

Reply:
The first error is:

```
init-tools.sh: line 9: __BUILDTOOLS_SOURCE: unbound variable
```

I'll give fixing it a try (waiting for a different build to finish).

Reply:
I wonder if we should just remove the "u" option in the root script.

Reply:
Personally, I really like how `-u` forces scripts to be clear if they expect the variable to be set at that point in time, and I think it's worth keeping. I haven't done a ton of work with it yet, though.